config/routes.rb

```diff
@@ -9,7 +8,11 @@
     confirmations: 'users/confirmations'
   }
 
-  resources :users, except: [:new, :create]
+  resources :events, only: [:index, :show]
 
-  get '*path'       , to: 'welcome#index'
```

Comment:
why this one is deleted? I think can put under namespace admin as the last safety precaution

Reply:
This causes problem when we want to access active storage data, because when we go to the url, it will be caught by WelcomeController. For now, I have not had any better idea other than deleting this route. Anyway, what do you mean by putting it under namespace admin? did you mean to just get 'admin/*path'?

Reply:
I mean as the last route, anyway, I see ... hmm then when will it go if you enter an invalid url?

Reply:
Routing Error Page